packages/core/src/modules/connections/repository/ConnectionMetadataTypes.ts

```diff
@@ -1,9 +1,13 @@
+import type { ConnectionType } from '../models'
+
 export enum ConnectionMetadataKeys {
   UseDidKeysForProtocol = '_internal/useDidKeysForProtocol',
+  ConnectionTypes = '_internal/connectionTypes',
 }
 
 export type ConnectionMetadata = {
   [ConnectionMetadataKeys.UseDidKeysForProtocol]: {
     [protocolUri: string]: boolean
   }
+  [ConnectionMetadataKeys.ConnectionTypes]: Array<ConnectionType | string>
```

Comment:
@TimoGlastra Should we just remove this for now?

Reply:
Yes, let's remove it if it isn't being used